cavityDisplayGUI.py

```diff
@@ -96,13 +98,19 @@ def __init__(self, parent=None, args=None):
                     # .add_callback is called when statusPV changes value
                     statusPV.add_callback(partial(self.statusCallback, cavityWidget))
 
+                    # .add_callback is called when descriptionPV
+                    descriptionPV.add_callback(partial(self.descriptionCallback, cavityWidget))
+
     # Updates shape depending on pv value
     def severityCallback(self, cavity_widget, value, **kw):
         self.changeShape(cavity_widget,
                          SHAPE_PARAMETER_DICT[value]
                          if value in SHAPE_PARAMETER_DICT
                          else SHAPE_PARAMETER_DICT[3])
 
+    def descriptionCallback(self, cavity_widget: QWidget, value, **kw):
```

Comment:
```suggestion
    @staticmethod
    def descriptionCallback(cavity_widget: QWidget, value, **kw):
```

Reply:
Wow, I should have looked through the git comments before bulldozing away at code!